examples/test_psf3d.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
```

Comment:
For `examples/test_psf3d.py`:
- remove numpy import
- add one line docstring at the top what this is
- Put `print(psf.info())` also before doing the plots.